src/app/events/[slug]/components/EventsSection.tsx

```diff
@@ -0,0 +1,21 @@
+import type { Event } from '@/types/eventType'
```

Comment:
I get this error here, not sure why

Cannot find module '@/types/eventType' or its corresponding type declarations.ts(2307)